netebpfext/net_ebpf_ext.c

```diff
@@ -43,48 +43,67 @@
 #include "ebpf_program_types.h"
 #include "ebpf_windows.h"
 #include "ebpf_xdp_program_data.h"
+#include "ebpf_flow_program_data.h"
+#include "ebpf_mac_program_data.h"
 
 static ebpf_ext_attach_hook_provider_registration_t* _ebpf_xdp_hook_provider_registration = NULL;
 static ebpf_ext_attach_hook_provider_registration_t* _ebpf_bind_hook_provider_registration = NULL;
+static ebpf_ext_attach_hook_provider_registration_t* _ebpf_flow_hook_provider_registration = NULL;
+static ebpf_ext_attach_hook_provider_registration_t* _ebpf_mac_hook_provider_registration = NULL;
 static ebpf_extension_provider_t* _ebpf_xdp_program_info_provider = NULL;
 static ebpf_extension_provider_t* _ebpf_bind_program_info_provider = NULL;
+static ebpf_extension_provider_t* _ebpf_flow_program_info_provider = NULL;
+static ebpf_extension_provider_t* _ebpf_mac_program_info_provider = NULL;
 
 #define RTL_COUNT_OF(arr) (sizeof(arr) / sizeof(arr[0]))
 
 #define NET_EBPF_EXTENSION_NPI_PROVIDER_VERSION 0
 
+
+// XDP Hook
 static ebpf_context_descriptor_t _ebpf_xdp_context_descriptor = {
     sizeof(xdp_md_t),
     EBPF_OFFSET_OF(xdp_md_t, data),
     EBPF_OFFSET_OF(xdp_md_t, data_end),
     EBPF_OFFSET_OF(xdp_md_t, data_meta)};
 static ebpf_program_info_t _ebpf_xdp_program_info = {{"xdp", &_ebpf_xdp_context_descriptor, {0}}, 0, NULL};
-
 static ebpf_program_data_t _ebpf_xdp_program_data = {&_ebpf_xdp_program_info, NULL};
-
 static ebpf_extension_data_t _ebpf_xdp_program_info_provider_data = {
     NET_EBPF_EXTENSION_NPI_PROVIDER_VERSION, sizeof(_ebpf_xdp_program_data), &_ebpf_xdp_program_data};
 
+// Bind Hook
```

Comment:
unnecessary change

Reply:
Did you mean the comments and spacing changes?

Reply:
yes